src/firstvoices/backend/fixtures/partsOfSpeech_initial.json

```diff
@@ -261,5 +261,69 @@
       "title": "Modifier",
       "parent": null
     }
+  },
+  {
+    "model": "backend.partofspeech",
+    "fields": {
+      "title": "Name",
+      "parent": null
+    }
+  },
+  {
+    "model": "backend.partofspeech",
+    "fields": {
+      "title": "Reflexive Verb",
+      "parent": [
+        "Verb"
+      ]
+    }
+  },
+  {
+    "model": "backend.partofspeech",
+    "fields": {
+      "title": "Entity",
+      "parent": null
+    }
+  },
+  {
+    "model": "backend.partofspeech",
+    "fields": {
+      "title": "Word Piece/Affix",
+      "parent": null
+    }
+  },
+  {
+    "model": "backend.partofspeech",
+    "fields": {
+      "title": "Event/Activity",
```

Comment:
From the wiki page it looks like this should be "Event/Activity (Verb-like word)"

Reply:
Also on the wiki page this lists itself as a parent... I wonder if the parent is meant to be "Verb"?

Reply:
Oh yes, this does not have a parent. Have confirmed from @caforbes and have updated the wiki page as well.